### PR TITLE
Merge master into lite

### PR DESCRIPTION
**Pull request by calvinjia**
Changes are in the last two commits.

### CODE REVIEW

File: core/common/src/main/java/alluxio/metrics/MetricKey.java [hunk 1]
```diff
@@ -773,6 +773,28 @@ public MetricKey build() {
           .setMetricType(MetricType.COUNTER)
           .setIsClusterAggregated(true)
           .build();
+  // Client local cache metrics
+  public static final MetricKey CACHE_BYTES_READ_CACHE =
```
Comment (LuQQiu) [inline core/common/src/main/java/alluxio/metrics/MetricKey.java]: Can you add a `CLIENT_` prefix to all those metric names to be consistent?

**Comment by calvinjia** [comment]
jenkins, test this please.

**Comment by calvinjia** [comment]
alluxio-bot, merge this please.

**Comment by calvinjia** [comment]
alluxio-bot, sync-merge this please.

**Comment by calvinjia** [comment]
Going to merge this manually since alluxio-bot won't do branch merge.

**Pull request closed without merging**